.gitignore

```diff
@@ -55,6 +54,7 @@ htmlcov/
 nosetests.xml
 coverage.xml
 *.cover
+*.py,cover
```

Comment:
Is this comma meant to be a `.`?

Reply:
I was just copying from https://github.com/github/gitignore/blob/master/Python.gitignore

Apparently it's right: https://github.com/github/gitignore/pull/3131